### ISSUE TITLE
Counting all data-team editions

### ISSUE THREAD

**Issue by Rub21** [mapbox/osm-edit-report-deprecated#65]
Last week I've spent some hours doing some counter for the team, basically, this will count w/ high detail the progres of each of the data-team member.  right now this si alive https://osm-edit-report-production.mapbox.com/json/m&1483833600&1484524800, running with osm-edit-report on the same server, and it is assets the number of roads which touched by the data-team. 

- The counter is updating every hour.
- We can make consult by hour, day, month and year. 
- Easily add  more tags  to count  we just need to add in https://s3.amazonaws.com/tofix/osmEditReport/tags.json

I think this could help the number of edition and objects made by data-team, along with osm-metrics,  I this case, we don't need to run any query. just add in the s3 file `tags.json` the tags.

**What is missing here?**

- A front-end and a good way to display all the numbers. 

cc. @ramyaragupathy @arunasank @planemad @maning

**Comment by Rub21** [comment]
This will really helpful for Peru team to track our progress as we are doing everyday mapping work. 
cc. @samely

**Comment by planemad** [comment]
<img width="203" alt="screenshot 2017-01-17 01 06 30" src="https://cloud.githubusercontent.com/assets/126868/21996634/3f3021a4-dc51-11e6-9043-41373bd3871a.png">

@Rub21 such statistics would be a good addition to osm-edit-report dashboard. Is this more reliable than the to-fix statistics?

**Comment by Rub21** [comment]
@planemad , Yes this is much better than the existing but is missing the front-end, I would like to take to build the front-end using react, I have been waiting to implement a small project such of this to learn react. However, if someone with high knowledge in React can do this faster would be great as well.

cc. @ajithranka @rasagy

**Comment by ajithranka** [comment]
Sounds great, @Rub21! Feel free to pull in me or @kepta if you'd like to pair with us.

**Comment by geohacker** [comment]
@Rub21 this will be super helpful!
